docs/user_guide/extending.rst

```diff
@@ -0,0 +1,188 @@
+            :code: rst
+            :class: highlight-rst
+
+This theme defines classes for `the standard docutils admonition types <https://docutils.sourceforge.io/docs/ref/rst/directives.html#admonitions>`__ (``attention``, ``caution``, etc) and additionally supports ``seealso`` and ``todo`` admonitions (see :doc:`../examples/kitchen-sink/admonitions` for a demo of all built-in admonition styles).
+
+
+Customizing the color
+---------------------
+
+Besides the pre-defined semantic color classes (see previous section) you can also add a bespoke color to any admonition by defining your own CSS class. Example:
+
+.. begin-example-color
+.. admonition:: Admonition with custom "olive" color
+    :class: admonition-olive
+
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+.. end-example-color
+
+Add the new class to your `custom.css <https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-html_css_files>`__ file. As in the example below, be sure to use the same color for ``border-color``, ``background-color``, and ``color`` (the transparency effect is handled automatically by the theme).
+
+.. tab-set::
+
+    .. tab-item:: rst
+
+        .. include:: ./extending.rst
+            :start-after: begin-example-color
+            :end-before: .. end-example-color
+            :code: rst
+            :class: highlight-rst
+
+    .. tab-item:: css
+
+        .. include:: ../_static/custom.css
+            :start-after: begin-custom-color
+            :end-before: /* end-custom-color
+            :code: css
+            :class: highlight-css
+
+
+Using a custom icon
+-------------------
+
+Customizing the icon uses a similar process to customizing the color: create a new CSS class in your `custom.css <https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-html_css_files>`__ file. The theme supports `fontawesome v6 icons <https://fontawesome.com/v6/search?o=r&m=free&f=brands>`__ ("free" and "brands" sets). To use an icon, copy its unicode value into your custom class as shown in the CSS tab below:
+
+.. begin-example-icon
+.. admonition:: Check out my custom icon
+    :class: admonition-icon
+
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+.. end-example-icon
+
+.. tab-set::
+
+    .. tab-item:: rst
+
+        .. include:: ./extending.rst
+            :start-after: begin-example-icon
+            :end-before: .. end-example-icon
+            :code: rst
+            :class: highlight-rst
+
+    .. tab-item:: css
+
+        .. include:: ../_static/custom.css
+            :start-after: begin-custom-icon
+            :end-before: /* end-custom-icon
+            :code: css
+            :class: highlight-css
+
+
+Combining all three customizations
+----------------------------------
+
+Here we demonstrate an admonition with a custom icon, color, and title (and also make it collapsible). Note that the multiple admonition class names are space-separated:
+
+.. begin-example-youtube
```

Comment:
I actually created a tiny little sphinx extension to show off syntax examples, maybe it would be useful here?

https://ebp-sphinx-examples.readthedocs.io/en/latest/

It gives you an example directive like:

````
```{example}
**foo**
```
````

And it create a little preview + the raw syntax to show how to make the preview.

Reply:
that would be perfect but you can't display 2 files yet right (like in these examples css + rst) ?

I will merge it and we'll see how to use it in the upcoming future